generator/abstractmetalang.cpp

```diff
@@ -657,6 +661,8 @@ QString AbstractMetaFunction::minimalSignature() const
     minimalSignature += ")";
     if (isConstant())
         minimalSignature += "const";
+    if (isConstexpr())
+        minimalSignature += "constexpr";
```

Comment:
For methods (and functions in general) const and constexpr mean completely different things; (const (f)()) says the result of f() may not be modified (so is pretty much a nonsense, since the result is always an r-value), (constexpr (f)()) says that f() gives the same result every time.  In practice 'const' will be in the return type whereas 'constexpr' modifies/qualifies the underlying function like a storage class (same as 'const' on a variable.)

Still I'm not sure it's correct either; it's just necessary to at least skip constexpr for Qt6.5.  Indeed I may have put the handling in the wrong place, or in too few places (it's not in the TypeInfo handling) but that's only going to become clear when enough of Qt6.5 (or, probably, Qt6.6 now) can be preprocessed into generated_cpp files that the compiler will accept!